lib/Parser/Legacy/NumberWithUnits.pm

```diff
@@ -68,7 +115,14 @@ sub getUnitNames {
 #
 sub getUnits {
   my $units = shift;
-  my %Units = Units::evaluate_units($units);
+  my $options = {};
+  if ($fundamental_units) {
+    $options->{fundamental_units} = $fundamental_units;
+  }
+  if ($known_units) {
+    $options->{known_units} = $known_units;
+  }
+  my %Units = Units::evaluate_units($units,{fundamental_units => $fundamental_units, known_units => $known_units});
```

Comment:
Good catch.  I was adding and removing stuff for testing purposes and likely added the wrong thing back in.